manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.zh-CN.yaml

```
@@ -0,0 +1,40 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+
+PackageIdentifier: GLPI-Project.GLPI-Agent
+PackageVersion: "1.11"
+PackageLocale: zh-CN
+Publisher: Teclib'
+PublisherUrl: https://glpi-project.org/
+PublisherSupportUrl: https://github.com/glpi-project/glpi-agent/issues
+PrivacyUrl: https://glpi-project.org/privacy-policy/
+Author: Teclib'
+PackageName: GLPI Agent
+PackageUrl: https://glpi-project.org/downloads/

```

Package URL is wrong as it only provides server part downloads, not agent related one. Better use the github release page.

```suggestion
PackageUrl: https://github.com/glpi-project/glpi-agent/releases
```